src/main/webapp/about.html

```diff
@@ -420,9 +420,12 @@ <h1 id="caption">About Us</h1>
                         <li>Paramsothy Sanshayan [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=sanshai28" target="_blank">bug reporting/fixing, enhancements</a>]</li>
                         <li>Prithviraj Billa [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=prithviraj.m.billa" target="_blank">bug reporting/fixing, enhancements</a>]</li>
                         <li>Sergey Belyablya [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=belyabl9" target="_blank">bug reporting/fixing, enhancements</a>]</li>
+                        <li>Shiv Kandikuppa [<a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Ashiv12095" target="_blank">bug reporting/fixing, enhancements</a>]</li>
                         <li>Steven Khong [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=khongwh" target="_blank">bug reporting/fixing, enhancements</a>]</li>
+                        <li>Sujeet [<a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Asujeet14108" target="_blank">bug reporting/fixing, enhancements</a>]</li>
                         <li>Tarun Bansal [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=tarunbansal.me" target="_blank">bug reporting/fixing, enhancements</a>]</li>
                         <li>Vaibhav Desai [<a href="https://code.google.com/p/teammatespes/issues/list?can=1&q=desaivaibhav94" target="_blank">bug reporting/fixing, enhancements</a>]</li>
+                        <li>Zhu Liang [<a href="https://github.com/TEAMMATES/teammates/issues?q=involves%3Aparadite" target="_blank">bug reporting/fixing, enhancements</a>]</li>
```

Comment:
May be you can do these at the same time? 
1. change all to 'involves'?
2. line wrapping for the whole section?

Can go ahead and merge after that.